stylus/ui-components/alerts.styl

```diff
@@ -30,6 +29,7 @@ $alerts
 
     .coz-alert
         display          block
+        z-index          100
```

Comment:
What changed since the last version of alerter that explains this new z-index? Unless it's a bugfix, maybe?